realm/realm-library/src/main/cpp/io_realm_RealmFileUserStore.cpp

```diff
@@ -41,6 +40,22 @@ Java_io_realm_RealmFileUserStore_nativeGetCurrentUser (JNIEnv *env, jclass)
     return nullptr;
 }
 
+JNIEXPORT jstring JNICALL
+Java_io_realm_RealmFileUserStore_nativeGetUser (JNIEnv *env, jclass, jstring identity)
+{
+    TR_ENTER()
+    try {
+        JStringAccessor id(env, identity); // throws
+        const std::shared_ptr<SyncUser> &user = SyncManager::shared().get_existing_logged_in_user(id);
```

Comment:
`std::shared_ptr<SyncUser>& user` no space between type name and `&`. See
https://github.com/realm/realm-core/blob/master/doc/development/coding_style_guide.cpp#L76